src/commands/ext-update.ts

```diff
@@ -203,7 +203,6 @@ export default new Command("ext:update <extensionInstanceId> [localDirectoryOrUr
         source: newSource,
         rolesToAdd: _.get(newSpec, "roles", []),
```

Comment:
`rolesToAdd` and `rolesToRemove` are both unnecessary now, and `update` method needs to be revised to no longer have them as parameters

Reply:
lines 194 to 198 also need to be removed.